concourse/pipeline/guest-package-build.yaml

```diff
@@ -0,0 +1,355 @@
+---
+resource_types:
+- name: gcs
+  type: registry-image
+  source:
+    repository: frodenas/gcs-resource
+
+resources:
+# centos
+- name: guest-agent
+  type: git
+  source:
+    uri: https://github.com/GoogleCloudPlatform/guest-agent.git
+    branch: master
+- name: guest-oslogin
+  type: git
+  source:
+    uri: https://github.com/GoogleCloudPlatform/guest-oslogin.git
+    branch: master
+- name: osconfig
+  type: git
+  source:
+    uri: https://github.com/GoogleCloudPlatform/osconfig.git
+    branch: master
+- name: guest-diskexpand
+  type: git
+  source:
+    uri: https://github.com/GoogleCloudPlatform/guest-diskexpand.git
+    branch: master
+- name: guest-configs
+  type: git
+  source:
+    uri: https://github.com/GoogleCloudPlatform/guest-configs.git
+    branch: master
+- name: guest-test-infra
+  type: git
+  source:
+    uri: https://github.com/GoogleCloudPlatform/guest-test-infra.git
+    branch: master
+
+jobs:
+- name: guest-agent-integration-test
+  plan:
+  - get: guest-agent
+  - get: guest-test-infra
+  - task: get-credential
+    file: guest-test-infra/concourse/tasks/get-credential.yaml
+  - task: guest-package-build-guest-agent
+    file: guest-test-infra/concourse/tasks/package-integration-test.yaml
+    vars:
+      repo-name: "guest-agent"
+- name: guest-oslogin-integration-test
+  plan:
+  - get: guest-oslogin
+  - get: guest-test-infra
+  - task: get-credential
+    file: guest-test-infra/concourse/tasks/get-credential.yaml
+  - task: guest-package-build-guest-oslogin
+    file: guest-test-infra/concourse/tasks/package-integration-test.yaml
+    vars:
+      repo-name: "guest-oslogin"
+- name: osconfig-integration-test
```

Comment:
there are existing tests but not written in gointegtest style; we'll have to work with andrew to see what we want here

Reply:
Sure.

Reply:
remove this job.